Makefile

```diff
@@ -128,12 +128,12 @@ node-check:
 		exit 1; \
 	fi
 
-.PHONY: clean-all
-clean-all: clean
+.PHONY: clean
+clean: clean-go
 	rm -rf $(JS_DEST_DIR) $(CSS_DEST_DIR)
 
-.PHONY: clean
```

Comment:
this will change expected default behaviour - should we realy change this?

Reply:
@silverwind @zeripath @sapk :arrow_up:

Reply:
It's mostly a personal preference so I can run `make clean build` to refresh all files. If people prefer `clean` to keep only cleaning build/integration, I can remove it again. I think @guillep2k had an opinion on it too.

Also, I'm contemplating whether `clean` should also remove `node_modules`.

Reply:
If `clean-go` is unneccesary, I'm happy to reduce it to just a single `clean` target.

Reply:
I normaly use `clean` as it works now ...

Reply:
Reverted it, I guess we can have that discussion another time :)